code/go/0chain.net/chaincore/chain/state.go

```diff
@@ -327,7 +327,7 @@ func (c *Chain) updateState(ctx context.Context, b *block.Block, bState util.Mer
 				zap.String("begin client state", util.ToHex(startRoot)),
 				zap.String("prev block", b.PrevBlock.Hash),
 				zap.Duration("time_spent", time.Since(t)),
-				zap.Any("txn", txn))
+				zap.String("txn", txn.Hash))
```

Comment:
Please log the txn as it can be helpful when we need to debug, same as the other places for txn below.